### ISSUE TITLE
usage of  image in extract_faces method

### ISSUE THREAD

**Issue by diego0718** [serengil/deepface#844]
Hi.
in extract_faces function over functions.py script we have this piece of code: 

img_pixels = image.img_to_array(current_img)
img_pixels = np.expand_dims(img_pixels, axis=0)
img_pixels /= 255  # normalize input in [0, 1]

why we need image  function for tensorflow.keras.preprocessing?. why not use np.array or something like this? .  For the moment i am only testing yunet detector

**Comment by serengil** [comment]
Thas was my choice, not a must

**Comment by diego0718** [comment]
I am grateful for fast reply!. 
Great work
